Tilix4/OpenPype
# Feature: build workfile fabrication

## Description
Add build fabrication workfile,
- Import all wanted subsets of LightSetupBank file and load concept reference if exists.
- Create 3 OpenPype instances ( cameraMain, setdressMain, reviewMain ). 
- Set camera parameters (Y location = -100, lens = 200), 
- Control DOF of camera using empty object, and apply the imported world, add the world to setdress instance.
- Create SET, PROPS, and CHARACTERS collections
- use `apply_settings` function, link to the PR : https://github.com/Tilix4/OpenPype/pull/49

## Testing:
1. Get the right version of normaal_addon : https://gitlab.com/NORMAAL/openpype-studio-addon/-/merge_requests/63
2. Use `ft_build_workfile_fabrication_test` branch to run the test
3. Launch an environment (fabrication)
4. Use build first workfile with `clear current scene` checked and `save as` unchecked
5. You should get the behavior described above
